stratuslab/java/jar/src/main/java/com/sixsq/slipstream/connector/stratuslab/StratusLabConnector.java

```diff
@@ -163,13 +160,7 @@ protected void validateLaunch(Run run, User user) throws ValidationException {
 		super.validateLaunch(run, user);
 
 		if (run.getCategory() == ModuleCategory.Image) {
-			ImageModule image = ImageModule.load(run.getModuleResourceUrl());
-			validateImageModule(image, user);
-		}
-		if (run.getCategory() == ModuleCategory.Deployment) {
-			for (Node node : DeploymentModule.load(run.getModuleResourceUrl()).getNodes().values()) {
-				validateImageModule(node.getImage(), user);
```

Comment:
It's because:
- getInstanceType method take a run as argument and should be a Simple Run. 
- I don't like the idea to check deployment in the CLI part. 
- This is an extra validation step, and it's the only connector doing that kind of validation.
- This need development time and it's a dying connector.